docs/design/design_v1.10.md

```diff
@@ -52,7 +52,7 @@ This means we aim to standardize:
  - Names of certificates files:
    - `ca.crt`, `ca.key` (CA certificate)
    - `apiserver.crt`, `apiserver.key` (API server certificate)
-   - `apiserver-kubelet-client.crt`, `apiserver-kubelet-client.key` (client certificate for the apiservers to connect to the kubelets securely)
+   - `apiserver-kubelet-client.crt`, `apiserver-kubelet-client.key` (client certificate for the apiservers to connect to the kubelet securely)
```

Comment:
IMO, either both need to be plural or singular in this example.
apiservers -> apiserver.
[1]